### ISSUE TITLE
Only one molecule from the input SMILES file is processed...

### ISSUE THREAD

**Issue by UnixJunkie** [mcs07/MolVS#30]
Hello,
```
molvs standardize -i smi input.smi  -o smi -O output.smi
```
The input file has several molecules, the output a single one.
Is this the expected behavior?

**Comment by UnixJunkie** [comment]
I installed vi pip.
```
pip install molvs
```

**Comment by UnixJunkie** [comment]
this is probably the file that needs a fix:
https://github.com/mcs07/MolVS/blob/master/molvs/cli.py

**Comment by mcs07** [comment]
Yes that seems to be the case. Also it looks like SDF files will only process the first molecule.

**Comment by UnixJunkie** [comment]
also, output to sdf file is not supported (it creates a mol file instead)

**Comment by UnixJunkie** [comment]
I'll send a patch for some of these.

**Comment by sgujjaABM** [comment]
Hi, I am having the same issue with an SDF file. Wondering if there's a fix?
Thanks

**Comment by UnixJunkie** [comment]
I sent a PR with a fix.
Use the molVS that is inside of rdkit, then you should be good to go.
